elastic/eui
# [EuiInlineEdit] Add `title` to Read Mode Button

## Summary
This PR adds the `title` prop to the `EuiEmptyButton` displayed in read mode. This is being added to improve readability of long text that may be truncated due to its length. The `title` will show the value of inline edit or the `placeholder` (depending on which is being shown). 

**With a stored value**
<img width="1082" alt="image" src="https://github.com/elastic/eui/assets/40739624/353fad08-6b50-484f-84e8-bcfd64add73a">

---

**With `placeholder`**
<img width="1097" alt="image" src="https://github.com/elastic/eui/assets/40739624/608d45f9-3418-4fa9-8301-cddc47de6cbc">


## QA

1. [Visit staging](https://eui.elastic.co/pr_6966/#/forms/inline-edit#display-and-edit-basic-text) and hover over the inline edit  (read mode) button for any example. For the [placeholder example](https://eui.elastic.co/pr_6966/#/forms/inline-edit#display-and-edit-basic-text), the placeholder text should populate the title until a value is saved.

Remove or strikethrough items that do not apply to your PR.

### General checklist
